src/miscwrappers.cpp

```diff
@@ -522,40 +528,48 @@ syscall(long sys_num, ...)
 # endif // ifdef __x86_64__
 #endif // ifndef DISABLE_SYS_V_IPC
 # ifndef __aarch64__
+# ifndef __riscv
   case SYS_poll:
   {
     SYSCALL_GET_ARGS_3(struct pollfd *, fds, nfds_t, nfds, int, timeout);
     ret = poll(fds, nfds, timeout);
     break;
   }
+# endif
 # endif // ifdef __aarch64__
 # ifndef __aarch64__
+# ifndef __riscv
```

Comment:
Same as before.  The new syscall to replace it is 'epoll_create1'm and `__NR_epoll_create1` exists along with `SYS_epoll_create`.  So, let's add a FIXME comment, that we can do:
```
// FIXME:  Change to:
// #ifdef SYS_epoll_create
// ... [Original code] ...
// #elif defined( SYS_epoll_create1)
// ... [Read 'man epoll_create1' and translate to that function.
// #endif
```